uwblueprint/planet-read
# Sort UsersTable by name

## Notion ticket link



[Add ability to sort UsersTable by name](https://www.notion.so/uwblueprintexecs/Add-ability-to-sort-UsersTable-by-name-8d579d6c0abf405e93562d703c96e8a7)



## Implementation description

- on clicking the name column header, sort the users
- display the appropriate arrow sign in the name column header
- abstracted `ManageTranslators`/reviewers to `ManageUsers` due to repeat code developing



## Steps to test

1. login and go to `/admin`
2. manage translators. observe that the users start sorted. 
3. click the name header a bunch. observe that the users order changes and the arrow next to the name header changes too. observe that more network calls are not made.
4. try it for manage reviewers. flip between reviewers and translators.



## What should reviewers focus on?

- does it work
- does `ManageUsers` make sense
- note: the interpretations ↑==ascending or ↑==descending is pretty 50/50 based on a quick roommate test. I dont think we should get caught up over the exact interpretation of ↑ vs ↓, because of it's not what the user was expecting, they can click it immediately again

## Checklist
